libres/lib/analysis/stepwise.cpp

```diff
@@ -8,370 +8,356 @@
+                    matrix_iset(stepwise->beta, ivar, 0,
+                                matrix_iget(beta, avar, 0));
+                    avar++;
+                }
+            }
         }
-      }
-    }
-
 
-    matrix_free( beta );
-  }
+        matrix_free(beta);
+    }
 
-  matrix_free( X );
-  matrix_free( E );
-  matrix_free( Y );
-  return y_mean;
+    matrix_free(X);
+    matrix_free(E);
+    matrix_free(Y);
+    return y_mean;
 }
 
-
-
-
-static double stepwise_eval__( const stepwise_type * stepwise , const matrix_type * x ) {
-  return matrix_row_column_dot_product( x , 0 , stepwise->beta , 0 );
+static double stepwise_eval__(const stepwise_type *stepwise,
+                              const matrix_type *x) {
+    return matrix_row_column_dot_product(x, 0, stepwise->beta, 0);
 }
 
+static double stepwise_test_var(stepwise_type *stepwise, int test_var,
+                                int blocks) {
+    double prediction_error = 0;
 
+    bool_vector_iset(stepwise->active_set, test_var,
+                     true); // Temporarily activate this variable
+    {
 
-static double stepwise_test_var( stepwise_type * stepwise , int test_var , int blocks) {
-  double prediction_error = 0;
-
-  bool_vector_iset( stepwise->active_set , test_var , true );   // Temporarily activate this variable
-  {
-
-    int nvar                       = matrix_get_columns( stepwise->X0 );
-    int nsample                    = matrix_get_rows( stepwise->X0 );
-    int block_size                 = nsample / blocks;
-    bool_vector_type * active_rows = bool_vector_alloc( nsample, true );
-
-
-
-
-
-    /*True Cross-Validation: */
-    int * randperms     = (int*)util_calloc( nsample , sizeof * randperms );
-    for (int i=0; i < nsample; i++)
-      randperms[i] = i;
+        int nvar = matrix_get_columns(stepwise->X0);
+        int nsample = matrix_get_rows(stepwise->X0);
+        int block_size = nsample / blocks;
+        bool_vector_type *active_rows = bool_vector_alloc(nsample, true);
 
-    /* Randomly perturb ensemble indices */
-    rng_shuffle_int( stepwise->rng , randperms , nsample );
+        /*True Cross-Validation: */
+        int *randperms = (int *)util_calloc(nsample, sizeof *randperms);
+        for (int i = 0; i < nsample; i++)
+            randperms[i] = i;
 
+        /* Randomly perturb ensemble indices */
+        rng_shuffle_int(stepwise->rng, randperms, nsample);
 
-    for (int iblock = 0; iblock < blocks; iblock++) {
+        for (int iblock = 0; iblock < blocks; iblock++) {
 
-      int validation_start = iblock * block_size;
-      int validation_end   = validation_start + block_size - 1;
+            int validation_start = iblock * block_size;
+            int validation_end = validation_start + block_size - 1;
 
-      if (iblock == (blocks - 1))
-        validation_end = nsample - 1;
+            if (iblock == (blocks - 1))
+                validation_end = nsample - 1;
 
-      /*
+            /*
```

Comment:
The indentation of this comment looks a bit funky since indentation of the code has changed.